cmd/manager/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
+
+	"github.com/omeryahud/ssp-operator/pkg/apis"
+	"github.com/omeryahud/ssp-operator/pkg/controller"
```

Comment:
What about already starting using kubevirt package path? I know we should use `github.com/omeryahud/ssp-operator`, but in few weeks we will migrate it under kubevirt org and then we will have to change all paths.

Reply:
I think it depends on users of this package. Are we going to migrate to the 'kubevirt.io' path before the HCO will use this package? Wouldn't it cause problems with go vendor if the module package path is different from the repository?

Reply:
I agree, we should generate the operator to declare the `kubevirt.io/ssp-operator` module.
This would require developers to clone `github.com/omeryahud/ssp-operator` to `$GOPATH/src/kubevirt.io/ssp-operator` manually for a while, but it would save us a huge headache when migrating the operator to the kubevirt organization.
Vatsal, could you re-generate the operator? If I'm not mistaken, the only manual changes made here were to the makefile/gitignore that can be easily migrated over.

Reply:
`kubevirt.io/ssp-operator` seems correct way to do it, generated a new skeleton now